pyi.Dockerfile

```diff
@@ -115,14 +115,15 @@ RUN ./build_node_package.sh
 # built on newer Ubuntu because they require new clang (newer than available in GPROFILER_BUILDER's CentOS 7)
 # these are only relevant for modern kernels, so there's no real reason to build them on CentOS 7 anyway.
 FROM ubuntu${PYPERF_BUILDER_UBUNTU} AS bcc-build
+ARG STATICX=""
```

Comment:
No need to modify the staticx config for pyperf  - the idea is to make gprofiler build faster when modifying only python code. I don't want to change the build logic of pyperf itself, only of the pyinstaller of gprofiler.